mgit.json

```diff
@@ -42,5 +46,8 @@
     "@ckeditor/ckeditor5-upload": "ckeditor/ckeditor5-upload",
     "@ckeditor/ckeditor5-utils": "ckeditor/ckeditor5-utils",
     "@ckeditor/ckeditor5-widget": "ckeditor/ckeditor5-widget"
+  },
+  "overrideDirectoryNames": {
+    "@ckeditor/ckeditor-cloud-services-core": "ckeditor5-cloud-services-core"
```

Comment:
I don't lime the fact that we're renaming package which name starts with `ckeditor-` to `ckeditor5-`. In my original proposal in https://github.com/cksource/mgit2/issues/72 I proposed to have a different directory structure, where all the prefixes (`ckeditor5-*`) are stripped. So, according to my comment, this setting here should be:

```
 "overrideDirectoryNames": {
      "@ckeditor/ckeditor5-cloud-services-core": "cloud-services-core"
  }
```

So, the actual directory name should be `cloud-services-core`.